### PR TITLE
Merge cgroups and environment variables for the resource allocation table.

### PR DESCRIPTION
**Pull request by SgtCoin**
Adding in a few brief lines about cgroups in the storage provider (sealing workers) section. Added all the environment variables for overriding the resource allocation table in lotus configuration. I am not sure what all we need for cgroups as its a linux feature and not specifically a lotus feature.

### CODE REVIEW

File: content/en/docs/storage-providers/seal-workers.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ Default resource value table. Some of these values are _fairly_ conservative:
 
 The Unseal task has the same resource use as the PreCommit1 task.
 
+***Note: The default (and custom) configurations in the task resource table can be overridden using environment variables on a per worker basis. See the [environment variables]({{< relref "config#environment-variables" >}}) section for information. You can also gather these details using the `lotus-worker resources --default` command.
```
Comment (johnnymatthews) [inline content/en/docs/storage-providers/seal-workers.md:60]:
```suggestion
{{< alert icon="info" >}}
The default and custom configurations in the task resource table can be overridden using environment variables on a per worker basis. See the [environment variables]({{< relref "config#environment-variables" >}}) section for information. You can also gather these details using the `lotus-worker resources --default` command.
{{< /alert >}}
```

File: content/en/docs/storage-providers/seal-workers.md [hunk 2]
```diff
@@ -101,6 +103,10 @@ When comparing task priority:
 - If the above is equal, tasks are selected based on priorities in the table
 - If the above is equal, sectors with lower sector numbers are selected (this can optimize gas usage slightly when submitting messages to the chain)
 
+### Cgroups
+
+Cgroups (control groups) is a Linux Kernel feature that limits, accounts for, and isolates the resource usage of a collection of processes. If cgroups are in use on the host, the Lotus Worker will honor the cgroup memory limits configured on the host. 
+
```
Comment (johnnymatthews) [inline content/en/docs/storage-providers/seal-workers.md:109]:
```suggestion
### Control groups

Countrol groups (cgroups) is a Linux Kernel feature that limits, accounts for, and isolates the resource usage of a collection of processes. If cgroups are in use on the host, the Lotus Worker will honor the cgroup memory limits configured on the host. 
```

**Review by johnnymatthews — APPROVED** [review]

File: content/en/docs/set-up/configuration.md [hunk 3]
```diff
@@ -234,6 +234,358 @@ Variables specific to the _Lotus daemon_:
...
+
+* 32G environment:
+```
+AP_32G_BASE_MIN_MEMORY=1073741824
+AP_32G_GPU_UTILIZATION=0
+AP_32G_MAX_MEMORY=4294967296
+AP_32G_MAX_PARALLELISM=1
+AP_32G_MAX_PARALLELISM_GPU=0
+AP_32G_MIN_MEMORY=4294967296
+C1_32G_BASE_MIN_MEMORY=1073741824
```
Comment (johnnymatthews) [inline content/en/docs/set-up/configuration.md:247]: These things are super long. I'm gonna try and add them into a `<details>` tag.

Comment (SgtCoin) [reply content/en/docs/set-up/configuration.md:247]: I wasnt sure of the best way to format it, but a drop down is a great idea.

Comment (johnnymatthews) [reply content/en/docs/set-up/configuration.md:247]: Adds a `<details>` shortcode: https://github.com/filecoin-project/lotus-docs/pull/61

**Comment by johnnymatthews** [comment]
Aaaaand the build is failing because it can't find the shortcode. @SgtCoin can you merge `main` into this branch please? Don't think I can do it since it's your PR. That should sort this error.

File: content/en/docs/storage-providers/seal-workers.md [hunk 4]
```diff
@@ -101,6 +105,9 @@ When comparing task priority:
 - If the above is equal, tasks are selected based on priorities in the table
 - If the above is equal, sectors with lower sector numbers are selected (this can optimize gas usage slightly when submitting messages to the chain)
 
+### Control groups
+
+Countrol groups (cgroups) is a Linux Kernel feature that limits, accounts for, and isolates the resource usage of a collection of processes. If cgroups are in use on the host, the Lotus Worker will honor the cgroup memory limits configured on the host.
```
Comment (johnnymatthews) [inline content/en/docs/storage-providers/seal-workers.md:110]:
```suggestion
Countrol groups (cgroups) is a Linux Kernel feature that limits, accounts for, and isolates the resource usage of a collection of processes. If cgroups are in use on the host, the Lotus Worker will honor the cgroup memory limits configured on the host.

```

**Pull request merged**